### PR TITLE
Feature/84 fix tab order on tabs

### PR DESCRIPTION
**Pull request by wwalmnes**
Resolves https://github.com/nrkno/core-components/issues/84

### CODE REVIEW

File: packages/core-tabs/core-tabs.js [hunk 1]
```diff
@@ -52,6 +52,20 @@ addEvent(UUID, 'keydown', (event) => {
   }
 })
 
+// Use capture to support older versions of firefox
+addEvent(UUID, 'focus', (event) => {
+  if (event.target && event.target.parentElement.getAttribute('role') === 'tablist') {
+    queryAll(event.target.parentElement.children).forEach((tab) => tab.setAttribute('tabindex', '-1'))
+  }
+}, true)
```
Comment (eirikbacker) [inline packages/core-tabs/core-tabs.js]: Too risky, and why check for `event.target`? :) we should check for `UUID`

```
addEvent(UUID, 'focus', ({target}) => { // Make sure there is no tabstops when inside tablist
  const tablist = event.target.parentElement
  if (target.getAttribute('tab') && tablist.hasAttribute(UUID)) {
    queryAll(tablist.children).forEach((tab) => tab.setAttribute('tabindex', -1))
  }
}, true)
```

File: packages/core-tabs/core-tabs.js [hunk 2]
```diff
@@ -52,6 +52,20 @@ addEvent(UUID, 'keydown', (event) => {
...
+
+addEvent(UUID, 'blur', (event) => {
+  if (event.target && event.target.parentElement.getAttribute('role') === 'tablist') {
```
Comment (eirikbacker) [inline packages/core-tabs/core-tabs.js]: Same as above :)

File: packages/core-tabs/core-tabs.js [hunk 3]
```diff
@@ -52,6 +52,20 @@ addEvent(UUID, 'keydown', (event) => {
...
+    queryAll(event.target.parentElement.children).forEach((tab) =>
+      tab.getAttribute('aria-selected') === 'true' && tab.setAttribute('tabindex', '0'))
```
Comment (eirikbacker) [inline packages/core-tabs/core-tabs.js]: tab.setAttribute('tabindex', tab.getAttribute('aria-selected') === 'true' ? -1 : 0)

File: packages/core-tabs/core-tabs.js (hunk 1 above)
Comment (wwalmnes) [reply packages/core-tabs/core-tabs.js]: Yup! Will check for `UUID` but drop the check for `tab` as we have the requirement for tabs to be direct children.

**Pull request merged**